front/nuxt.config.js

```diff
@@ -154,5 +155,10 @@ module.exports = {
         })
       }
     }
+  },
+
+  sentry: {
+    dsn: process.env.SENTRY_DSN,
+    config: {}, // Additional config
```

Comment:
configure also uploading of sourcemaps to sentry

Reply:
In the documentation of Sentry and internet there's no clear decription of how to use sourcemaps with Nuxtjs or Vue. Do you know how to use it?

Reply:
searching by 'sourcemap' in /front finds results only from 'front/node_modules' . Do we use sourcemaps in the project?

Reply:
see https://github.com/nuxt-community/sentry-module/blob/7c064df4feefe27ff3c63c1940b36b648959c875/README.md#webpackconfig

source map are generated by nuxt via webpack devtool you need to enable it in nuxt config
https://nuxtjs.org/api/configuration-build/#extend